generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>Please update the output for this cell, should be outdated here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>